### DISCUSSION TITLE
How do I fix this?

### DISCUSSION THREAD

**Discussion by alliyaa** [community/community#114230] [Other Feature Feedback, Questions, & Ideas]
If anyone has encountered this before please help. I thought it was just my page but I tried opening a link and it's the same. I tried on different browsers and tried opening the link on my page and my profile on my phone too but the formatting is still weird. ![Uploading 316453613...]![image](https://github.com/community/community/assets/147240608/729f4be5-4213-4866-a40d-bb4b071a6531)

**Comment by pierpaolosestito-dev** [comment]
I try to visit your page but It tells to me that you doens't have any public gists yet. I thought that It was a problem maybe with the .css file that your browser should download, but maybe It doens't do it.
Try to see if you block any download of JavaScript and other files also.

**Reply by alliyaa** [reply]
I checked within site permissions and javascript is allowed ![image](https://github.com/community/community/assets/147240608/2ca861b4-a89b-4161-b520-dfe5094935ba)
